Sources/SwiftKuery/Column.swift

```diff
@@ -16,8 +16,26 @@
 
 // MARK: Column
 
-/// Definition of table column.
+/**
+ The `Column` class is used to represent a single column in an SQL table in Swift.
+ A combination of columns are used to construct a `Table` class which matches a specific table in an SQL database.
+ The `Column` class details the column name, the table the column belongs to, any SQL keywords which apply to the column and the data type of the column.
+ ### Usage Example: ###
+ In this example, a person `Table` class matching a table stored in an SQL database is defined.
+ The "PersonTable" class contains the table name and three instances of the `Column` class.
+ The person_id column is an set as an autoIncrementing, primary key, containing values of type Int32, which must be unique and not null.
+ The name column contains not null, Strings and monthlyPay contains values of type Int32.
+ ```swift
+ public class PersonTable : Table {
+     let tableName = "personTable"
+     let person_id = Column("person_id", Int32.self, autoIncrement: true, primaryKey: true, notNull: true, unique: true)
+     let name = Column("firstName", String.self, notNull: true)
+     let monthlyPay = Column("monthlyPay", Int32.self)
+ }
+ ```
+ */
 public class Column: Field, IndexColumn {
+    // MARK: Column Parameters
```

Comment:
Line 69 /// The expression to check for values inserted into of the column.
I don't know what this checkExpression parameter is for but this is not English.